src/mrpro/algorithms/optimizers/pgd.py

```diff
@@ -28,6 +28,7 @@ def pgd(
     stepsize: float = 1.0,
     max_iterations: int = 128,
     backtrack_factor: float = 1.0,
+    t_for_converging_solution: bool = False,
```

Comment:
convergent_iterates_variant?
convergent_iterates?

i would not mention "t" in the argument name